src/components/CommentSection/CommentInput.js

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 const CommentInput = props => {
+  const [com, setCom] = useState('');
+  const changeComment = e => {
+    setCom(e.target.value);
+  }
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    props.setComments(com)
```

Comment:
Nice attempt to add new comments.
To get this to function as intended, you will need to check the structure of comments in `dummy-data.js`

**Hint**: It is an object with keys `username` and `text` 

You might need to create an object of this nature based on the username of your choice and the value of `com` then `concat` it to the existing comments using `setComments`